compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -127,6 +127,176 @@ void overwrite_quantparam(luci::CircleNode *source, luci::CircleNode *target)
   target_qparam->quantized_dimension = source_qparam->quantized_dimension;
 }
 
+/**
+ * tells if pad_v2 quantization should ignore padding value
+ * In that case padding const will be quantized with input parameters, and probably clipped
+ */
+bool ignore_pad_v2_const_quantization(luci::CirclePadV2 *pad)
+{
+  // This is a workaround to quantize pad generated from MaxPoolWithArgmax operation properly
+  // TODO use metadata hints to detect this case
+  auto const_value_node = dynamic_cast<luci::CircleConst *>(pad->arg(2));
```

Comment:
and `<const luci::CircleConst *>` too?

Reply:
I've locally checked that `dynamic_cast<const luci::CircleConst *>` is ok.